.vscode/settings.json

```diff
@@ -17,5 +17,6 @@
     "**/obj": true,
     "**/publish/**": true,
     "**/Debug": true
-  }
+  },
+  "typescript.surveys.enabled": false
```

Comment:
What does this do?

Reply:
In the newest Visual Studio Code you're asked to survey, I said no. And it saved it in the workspace settings.